docs/compiling-a-contract.rst

```diff
@@ -7,7 +7,7 @@ To compile a contract, use:
 
 You can also compile to other formats such as ABI using the below format:
 ::
-    vyper -f ['abi', 'json', 'bytecode', 'bytecode_runtime', 'ir'] yourFileName.vy
+    vyper -f ['python_abi', 'abi', 'bytecode', 'bytecode_runtime', 'ir', 'asm'] yourFileName.vy
```

Comment:
while i was at it, I noticed that `asm` was missing from the docs as well, i fixed that as well

is there a good way to force couple the code with these docs?

Reply:
no easy way that I can think of right away, no.